Deprecated warning when compiling with ESP-IDF 4.x.

The following warning occurs when compiling with ESP-IDF 4.x.

```
In file included from .pio/libdeps/m5paper-test/LovyanGFX/src/utility/miniz.c:959:
.pio/packages/framework-arduinoespressif32/tools/sdk/esp32/include/esp32/include/esp_spiram.h:1:2: warning: #warning esp_spiram.h has been replaced by esp32/spiram.h, please include esp32/spiram.h instead [-Wcpp]
 #warning esp_spiram.h has been replaced by esp32/spiram.h, please include esp32/spiram.h instead
```

This is because `#include "esp_spiram.h"` was deprecated in ESP-IDF 4.0.
I'll create a PR to fix this issue.

--- a/src/utility/miniz.c
+++ b/src/utility/miniz.c
@@ -956,7 +956,11 @@ typedef unsigned char mz_validate_uint64[sizeof(mz_uint64)==8 ? 1 : -1];
   #define MZ_REALLOC(p, x) NULL
 #else
   #ifdef BOARD_HAS_PSRAM
-    #include "esp_spiram.h"
+    #if __has_include(<esp32/spiram.h>)
+      #include <esp32/spiram.h>
+    #else
+      #include <esp_spiram.h>
+    #endif
     #include "soc/efuse_reg.h"
     #include "esp_heap_caps.h"
     #define MZ_MALLOC(x) heap_caps_malloc(x, MALLOC_CAP_SPIRAM | MALLOC_CAP_8BIT)

